apps/nextjs-website/src/components/molecules/SolutionStep/SolutionStep.tsx

```diff
@@ -9,12 +9,18 @@ type SolutionStepProps = {
   title: string;
   content: BlocksContent;
   products: { label: string; href: string }[];
+  width?: string;
 };
 
-const SolutionStep = ({ title, content, products }: SolutionStepProps) => {
+const SolutionStep = ({
+  title,
+  content,
+  products,
+  width,
+}: SolutionStepProps) => {
   const t = useTranslations('solution');
   return (
-    <Stack flexBasis={{ lg: '334px' }} gap={2} justifyContent='space-between'>
+    <Stack sx={{ width }} gap={2} justifyContent='space-between'>
```

Comment:
I think that width should be applied only for large displays
![image](https://github.com/pagopa/developer-portal/assets/39835990/26a7916f-9586-4146-8756-45596ef5f12d)